staging/src/k8s.io/cri-api/pkg/apis/runtime/v1alpha2/api.proto

```diff
@@ -233,6 +244,12 @@ message NamespaceOption {
     // Note: There is currently no way to set CONTAINER scoped IPC in the Kubernetes API.
     // Namespaces currently set by the kubelet: POD, NODE
     NamespaceMode ipc = 3;
+    // User namespace for this container/sandbox.
+    // Note: There is currently no way to set CONTAINER scoped user namespace in the Kubernetes API.
+    // The container runtime should ignore this if user namespace is NOT enabled.
```

Comment:
Should it really ignore that? Isn't it a security risk to schedule a pod / container when all the requirements for it are not met?

Reply:
That's seems strange indeed, I would rather return an error to make things clearer. On the other hand, this is not exposed to the user, so it is not possible to write a pod yaml which would generate `NamespaceOption.user = CONTAINER`, so it should not be a security risk.